llvm/include/llvm/DebugInfo/Symbolize/Symbolize.h

```diff
@@ -199,8 +199,14 @@ class LLVMSymbolizer {
   /// Return a pointer to object file at specified path, for a specified
   /// architecture (e.g. if path refers to a Mach-O universal binary, only one
   /// object file from it will be returned).
-  Expected<ObjectFile *> getOrCreateObject(const std::string &Path,
-                                           const std::string &ArchName);
+  Expected<ObjectFile *> getOrCreateObject(const std::string &InputPath,
+                                           const std::string &DefaultArchName);
+
+  /// Return a pointer to object file at specified path, for a specified
```

Comment:
```suggestion
  /// Return a pointer to the object file at specified path, for a specified
```
(the comment above also has this issue).

Reply:
added